modules/ROOT/pages/sap/sap-connector-reference-43.adoc

```diff
@@ -0,0 +1,802 @@
+|===
+
+[[outbound-config_simple-connection-provider]]
+=== Simple Connection Provider
+
+
+Connection provider that connects using username and password.
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| SAP system number a| String |  The two-digit system number (sysnr) of the SAP system. |  | x
+| SAP client ID a| String |  Three-digit client number. Preserve leading zeros if they appear in the number. |  | x
+| Destination Name a| String |  Identifier of the destination generated by this connector. If none is set, then a random value will be created. |  | 
+| Login language a| String |  Code of the language used by the connector. The language determines the code page used for communicating between SAP Adapter and the application server. |  EN | 
+| Extended Properties a| Object |  |  | 
+| SAP username a| String |  The username with which the connector will log into SAP. |  | x
+| SAP password a| String |  The password associated with the login username. |  | x
+| Application server host a| String |  The host of the SAP application server. |  | 
+| Message server configuration a| <<MessageServer>> |  The configuration for message server usage. |  | 
+| Enable RFC tracing on server a| Boolean |  Enable or disable RFC trace on the server. |  false | 
+| Enable CPIC tracing on server a| Enumeration, one of:
+
+** INHERIT
+** NO_TRACING
+** LEVEL_1
+** LEVEL_2
+** LEVEL_3 |  Enable or disable RFC trace on the server. |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+|===
+
+== Supported Operations
+
+* <<confirmTransactionId>> 
+* <<createIdoc>> 
+* <<createTransactionId>> 
+* <<executeQueuedRemoteFunctionCall>> 
+* <<executeSynchronousRemoteFunctionCall>> 
+* <<executeTransactionalRemoteFunctionCall>> 
+* <<getFunction>> 
+* <<send>> 
+
+== Operations
+
+[[confirmTransactionId]]
+=== Confirm Transaction ID
+`<sap:confirm-transaction-id>`
+
+
+Confirms a determined transaction.
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Transaction Id a| String |  The ID of the transaction to confirm. |  | x
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  The type of joining action that operations can take regarding transactions. |  JOIN_IF_POSSIBLE | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | 
+|===
+
+
+=== For Configurations
+* <<outbound-config>> 
+
+==== Throws
+* SAP:RETRY_EXHAUSTED 
+* SAP:CONNECTIVITY 
+
+
+[[createIdoc]]
+=== Create IDoc
+`<sap:create-idoc>`
+
+
+Retrieves a IDocument based on its key.
```

Comment:
From a quick we search, it appears that IDocument is an incorrect usage - should either be IDoc or intermediate Document — see https://en.wikipedia.org/wiki/IDoc